linker/Linker/Driver.cs

```diff
@@ -72,7 +74,71 @@ public static int Execute (string[] args, ILogger customLogger = null)
 
 		public Driver (string [] args)
 		{
-			_queue = new Queue<string> (args);
+			_queue = ProcessResponseFile (args);
+		}
+
+		Queue<String> ProcessResponseFile (string [] args)
+		{
+			var result = new Queue<string> ();
+			foreach (string arg in args) {
+				if (arg.StartsWith ("@")) {
```

Comment:
This logic is wrong, the response file value should override command line value if specified later

Reply:
As of now - the response file content are like macro and get expanded in place. 
It looks like some command line arguments, like `--skip-unresolved`, are meant to be specified once, even though it doesn't really matter if it is specified more than once. Some arguments, such as `-d`, are meant to be specified more than once, and it doesn't make sense to override.

How about just keep the current expansion logic, and if we need to implement the override, we can choose to do that at run?

Reply:
I'd rather not introduce uncommon behaviour which we cannot change once it lands

Reply:
Can you please elaborate on what do you mean by the override logic, maybe with some examples?

Reply:
Any updates?

Reply:
Consider someone passes following arguments `@foo.rsp -c skip` this should overwrite -c setting from foo.rsp

Reply:
Somehow I responded to this comment somewhere else, resolving this one and continue the conversation there instead.